pkg/providers/providers.go

```diff
@@ -22,6 +22,10 @@ var Storage storage.Storage
 
 type ProviderInitFunc func() error
 
+func init() {
+	NetworkPlugins = make(map[string]network.Plugin)
```

Comment:
This can be an inline initialization: `var NetworkPlugins = make(map[string]network.Plugin)`, no need for `init()`